modules/framePreProc/framePreProc.py

```diff
@@ -0,0 +1,48 @@
+
+THRESHHOLD = 30
+
+class FramePreProc:
+
+    """
+
+        Filters out useless telemetry data i.e data whose change in euler angles from the previous state exceeds 30 degrees
+
+        Attributes
+        ----------
+        eulerDict : dict
+            dictionary holding the eulerAnglesOfCamera data from telemetry
+        eulerDictLast : dict
+            dictionary holding the previous eulerAnglesOfCamera to compare to
+
+    """
+
+    def __init__(self, eulerDict, eulerDictLast):
+
+        self.eulerDict = eulerDict
+        self.eulerDictLast = eulerDictLast
+
+    def filter(self):
```

Comment:
Instead of storing self.eulerDict, pass in the current eulerDict as an argument.